TestModels/LanguageSpecificLogic/runtimes/net/LanguageSpecificLogic.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>LanguageSpecificLogic</RootNamespace>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <LangVersion>10</LangVersion>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DafnyRuntime" Version="4.0.0.50303"/>
```

Comment:
Question: `4.0.0`? Should we bump this?

This may be an issue spread through out the Test Models...
We should figure out a solution for this...

Reply:
Let's leave this as-is for now. It doesn't cause issues, and other TestModels use this version.
I would prefer a separate effort to bump the minimum Dafny version for Smithy-Dafny, since that may be more involved.

I see the next steps here as:
* Dafny publishes Java DafnyRuntime 4.4.0. This should unblock us to drop support for Dafny <4.4.0 from our Smithy-Dafny/MPL/DBESDK.
* We implement Java in this TestModel
* We bump Smithy-Dafny/TestModels minimum version to 4.4.0, upgrade NET/Java runtimes to 4.4.0, drop CI for <4.4.0
* We bump MPL/DBESDK minimum version to 4.4.0

At the end of this, we are now unblocked for per-language behavior in our libraries.

Reply:
https://sim.amazon.com/issues/CrypTool-5262